keps/prod-readiness/sig-apps/3998.yaml

```diff
@@ -3,3 +3,5 @@ alpha:
   approver: "@wojtek-t"
 beta:
   approver: "@wojtek-t"
+stable:
+  approver: "@wojtek-t"
```

Comment:
Hi, @wojtek-t!
Would you mind taking this graduation as PRR approver?
Thanks.

Reply:
Sorry - I was out for last 3 weeks. Taking a look now.

Reply:
No problem. Thanks for the review!